src/core/annotations/field-value.ts

```diff
@@ -22,13 +19,13 @@ function booleanFieldValueFactory() {
     const defaultValue = target[key];
 
     // Use a fallback if Symbol isn't available.
-    const localKey = isPresent(Symbol) ? Symbol(key) : `__md_private_symbol_${key}`;
+    const localKey = Symbol ? Symbol(key) : `__md_private_symbol_${key}`;
     target[localKey] = defaultValue;
 
     Object.defineProperty(target, key, {
       get() { return this[localKey]; },
       set(value: boolean) {
-        this[localKey] = isPresent(value) && value !== null && String(value) != 'false';
+        this[localKey] = value != null && `${value}` !== 'false';
```

Comment:
`value != false` isn't allowed by TypeScript (comparing string to boolean).

To me `${value}` is way cleaner than using `toString()` and more readable than `'' + value` (due to the boolean operator) while being the same number of characters).